tests/cache/models/aspects/runStandardTests.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/cache/models/aspects/runStandardTests.js
+ */
+'use strict'; // eslint-disable-line strict
+const tu = require('../../../testUtils');
+const requireDir = require('require-dir');
+
+/*
+ * This test runs the standard aspect api tests with the cache enabled.
+ * Note that this must be run in a separate command from the api tests,
+ * otherwise these tests will not run because files can't be required
+ * twice in the same process.
+ */
+
+describe('tests/cache/models/aspects/runStandardTests.js, ', () => {
+  before(() => tu.toggleOverride('enableRedisSampleStore', true));
+  after(() => tu.toggleOverride('enableRedisSampleStore', false));
+
+  describe('run standard tests with cache enabled', () => {
+    requireDir('../../../api/v1/aspects');
+  });
```

Comment:
Some of the related bugs I found with updating aspects were cache-specific, and weren’t being covered in the existing tests.
This is to set up the aspect tests to run with the cache enabled, like I did recently with subjects and samples. 
 I thought this wouldn’t be necessary before because all the tests were passing, but I realized it was because all the sample access in the aspect tests was going directly to the db. I changed it to use the api instead for all sample access.